### ISSUE TITLE
 2023.11.17-98cc15e outdated while 2023.11.15-06ab1ea latest

### ISSUE THREAD

**Issue by AJCxZ0** [0xpr03/nitter-status#5]
According to [Nitter Instance Uptime & Health](https://status.d420.de/) right now

| Instance | Country | Health History | All Time % | RSS | Nitter Version | Points |
|--------|--------|--------|--------|--------|--------|--------|
|  [nitter.unixfox.eu](https://nitter.unixfox.eu/) | 🇩🇪 | ✅ | 81% | ❌ | 2023.11.15-06ab1ea **latest** | 73 |
| [nitter.x86-64-unknown-linux-gnu.zip](https://nitter.x86-64-unknown-linux-gnu.zip/) | 🇩🇪 | ✅ | 78% | ✅ | 2023.11.17-98cc15e **outdated** | 73 |

**Comment by 0xpr03** [comment]
Yep, we're just validating what is the latest commit on the relevant branch right now.

To be precise there is another issue: The host marked as outdated is actually from a different branch, as you can see when clicking on the [commit](https://github.com/zedeus/nitter/commit/98cc15e) linked by unknown-linux-gnu.zip.

**Comment by AJCxZ0** [comment]
Thank you for addressing this issue, @0xpr03. Indeed I see

> ⚠ This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

which I interpret as differing from **foreign repo**.

If the **latest** and **outdated** tags refer only to the commits on a branch rather than the date which I (possibly incorrectly) interpreted as the version, then perhaps this information can be communicated more clearly than a single **Nitter Version** column while remaining sufficiently succinct to be useful. I hesitate to suggest a specific alternative, as my understanding of the available data is insufficient.

**Comment by 0xpr03** [comment]
> which I interpret as differing from foreign repo.

This happens if you fork the repo and make your own commits, but don't change the github repo URL. To me this is a foreign repo - it contains code that is not on the official nitter repo, so may have foreign functionality. (It is also not that easy to check whether this is built on top of the current version, or actually additional code based on an older one.)

**Comment by 0xpr03** [comment]
I think we should at least detect foreign commits. Not sure what to do about the other part.
